src/graphics/bargraphics.jl

```diff
@@ -4,21 +4,23 @@ mutable struct BarplotGraphics{R<:Number} <: GraphicsArea
     char_width::Int
     max_freq::Number
     max_len::Int
-    symb::String
+    symbols::AbstractVector{String}
     transform
 
     function BarplotGraphics(
             bars::AbstractVector{R},
             char_width::Int,
             color::UserColorType,
-            symb::Union{Char,String},
-            transform) where {R}
-        length(symb) == 1 || throw(ArgumentError("The symbol to print has to be a single character, got: \"" * symb * "\""))
+            symbols::Union{S},
+            transform) where {R, S <: Union{Char, String}}
```

Comment:
I was getting a crash trying to run
```julia
julia> using UnicodePlots

julia> UnicodePlots.histogram(randn(1000) * 0.1, closed = :right, nbins = 15)
```

until I did:
```diff
-            transform) where {R, S <: Union{Char, String}}
+            transform) where {R, S <: Union{Char, String, Vector{String}}}
```

But then I got
```
                  ┌                                        ┐
   (-0.35, -0.3 ] ┤▉ 2
   (-0.3 , -0.25] ┤█▏ 6
   (-0.25, -0.2 ] ┤██▎ 13
   (-0.2 , -0.15] ┤███████▎ 43
   (-0.15, -0.1 ] ┤██████████████████▏ 104
   (-0.1 , -0.05] ┤███████████████████████████▏ 153
   (-0.05,  0.0 ] ┤███████████████████████████████▏ 174
   ( 0.0 ,  0.05] ┤███████████████████████████████████▏ 194
   ( 0.05,  0.1 ] ┤████████████████████████████▏ 158
   ( 0.1 ,  0.15] ┤█████████████████▏ 97
   ( 0.15,  0.2 ] ┤█████▏ 30
   ( 0.2 ,  0.25] ┤███▎ 20
   ( 0.25,  0.3 ] ┤▉ 5
   ( 0.3 ,  0.35] ┤▉ 1
                  └                                        ┘
```
should the bars for 1, 2, 5, 6 be that close?

Reply:
Thanks for the feedback, yes that PR is not yet ready, needs more polishing, tests, ...
About the symbols being close, no I think my maths on this one are wrong :wink:

Reply:
Ah, ok, see my comment below. Hope it helps :)

Reply:
@aminnj, about the types (`Union{Char, String, Vector{String}}`) on what julia version are you ? 
It works with the original PR on my machine using `julia@1.6.2`, can't reproduce the crash ...

Reply:
I was using 1.6.0. I'll try again with 1.6.2 tomorrow and see if that fixes it.

Reply:
We'll see anyway if this PR passes on older releases in CI when I will have generated and pushed new ref imgs.